docs/area-owners.md

```diff
@@ -48,58 +48,54 @@ Below table shows the combined area owners on this repository:
 | area-Tracing-mono | @lambdageek | |
 | area-Performance-mono | @SamMonoRT | |
 | **System contract assemblies** | | |
-| area-System.AppContext | @safern @Anipik | |
```

Comment:
Was this removal intentional? Is it going into `System.Runtime`?

Reply:
Also, same questions for `area-System.ServiceModel` and `area-System.Runtime.Extensions`.

Reply:
Yes, I merged it into System.Runtime, there is not much value in having a separate area.

Reply:
ServiceModel never lived in our repo and had no relevant issues.  System.Runtime.Extensions isn't a namespace, it was an a assembly with a grab-bag of types which have now all been moved into System.Runtime.

Reply:
Ok, sounds good. Thanks 😄